### PR TITLE
6133 ui play video

### PR DESCRIPTION
**Pull request by dinhquang251**
https://edu-redmine.framgia.vn/issues/6133
Create play youtube UI
52637319_2003643629940727_886805515630280704_n

### CODE REVIEW

**Comment by dinhquang251** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_35/screen/youtube/YoutubeActivity.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+
+import com.framgia.moviedb_35.R;
+
+import static com.framgia.moviedb_35.screen.home.HomeViewModel.BUNDLE_KEY;
+import static com.framgia.moviedb_35.screen.home.HomeViewModel.BUNDLE_YOUTUBE_KEY;
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/screen/youtube/YoutubeActivity.java:12]: remove bl

Comment (NguyenMinhTuanB) [reply app/src/main/java/com/framgia/moviedb_35/screen/youtube/YoutubeActivity.java:12]: Ctr + ATL + L & Ctr + ATL + O format code tất cả các class trước khi gửi.

**Comment by dinhquang251** [comment]
ready

**Pull request merged**